.github/workflows/build_test.yml

```diff
@@ -4,19 +4,15 @@ on:
   # allows us to run workflows manually
   workflow_dispatch:
   pull_request:
-    paths-ignore:
-      - '.github/workflows/docker_publish.yml'
-      - '.github/workflows/changelog_test.yml'
-      - 'docker/**'
-      - 'CHANGELOG.rst'
+    paths:
+      - '.github/workflows/build_test.yml'
+      - '.github/actions/build-test/action.yml'
```

Comment:
This says it will only run if these files change, but we want testing to run for any file changes